web-components/src/components/icons/ProfileIcon.tsx

```diff
@@ -0,0 +1,79 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+import { sxToArray } from '../../utils/mui/sxToArray';
+
+interface ProfileIconProps extends SvgIconProps {}
+
+export default function ProfileIcon({
+  sx,
+  ...props
+}: ProfileIconProps): JSX.Element {
+  return (
+    <SvgIcon
+      sx={[{ color: '#4FB573' }, ...sxToArray(sx)]}
+      width="13"
+      height="13"
+      viewBox="0 0 13 13"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <svg
+        width="24"
+        height="24"
+        viewBox="0 0 24 24"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g clip-path="url(#clip0_251_24054)">
+          <g filter="url(#filter0_d_251_24054)">
+            <path
+              fill-rule="evenodd"
+              clip-rule="evenodd"
+              d="M21 4C21 2.34315 19.6569 1 18 1C16.3431 1 15 2.34315 15 4C15 4.13134 15.0084 4.26071 15.0248 4.38759L6.41901 9.35616C5.99653 9.12892 5.51332 9 5 9C3.34315 9 2 10.3431 2 12C2 13.6569 3.34315 15 5 15C5.56138 15 6.08675 14.8458 6.53606 14.5775L15.0433 19.4891C15.0148 19.6552 15 19.8258 15 20C15 21.6569 16.3431 23 18 23C19.6569 23 21 21.6569 21 20C21 18.3431 19.6569 17 18 17C17.2421 17 16.5498 17.2811 16.0217 17.7446L7.82495 13.0122C7.93827 12.696 8 12.3552 8 12C8 11.6033 7.923 11.2246 7.78313 10.878L15.9319 6.17327C16.4701 6.68555 17.1983 7 18 7C19.6569 7 21 5.65685 21 4Z"
+              fill="white"
+            />
+          </g>
+        </g>
+        <defs>
+          <filter
+            id="filter0_d_251_24054"
```

Comment:
FYI, if two SVGs share the same ID one of them will not display correctly because of ID conflict.
Here it should be safe, but it's important to double-check check id generated or even better rename it.

Reply:
i've renamed these, let me know if they don't look good for some reason